mobilecoind-json/tests/integration_test.py

```diff
@@ -57,8 +57,16 @@ def get_monitor(self, monitor_id):
     def get_ledger_local(self):
         return self.request(f"ledger/local")
 
-    def account_key_from_mnemonic(self, mnemonic):
-        return self.request("account-key-from-mnemonic", {"mnemonic": mnemonic})
+    # Takes the json loaded from a private key file, tests if it is mnemnonic or root entropy
+    #
+    # If it is mnemonic, we pass the mnemonic to `account-key-from-mnemonic` route
+    # Otherwise, we hit the `entropy/{root_entropy}` route.
+    # Both return Json account key response.
+    def account_key_from_json(self, obj):
+        if "mnemonic" in obj:
+            return self.request("account-key-from-mnemonic", {"mnemonic": obj["mnemonic"]})
+        else
+            return self.request("entropy/{}".format(obj["root_entropy"]), {})
```

Comment:
Consider checking for `root_entropy` and bailing if it doesn't exist, similar to #2017 

```suggestion
    def account_key_from_json(self, key_data):
        if "mnemonic" in key_data:
            return self.request("account-key-from-mnemonic", {"mnemonic": key_data["mnemonic"]})
        elif "root_entropy" in key_data:
            return self.request("entropy/{}".format(key_data["root_entropy"]), {})
        else:
            raise Exception("unknown key format")
```